src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/CheckCoscMarkPaidCallbackHandler.java

```diff
@@ -47,7 +47,7 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse checkIsMarkedPaid(CallbackParams callbackParams) {
 
         CaseData caseData = callbackParams.getCaseData();
-        CaseData.CaseDataBuilder dataBuilder = caseData.toBuilder();
+        CaseData.CaseDataBuilder dataBuilder = caseData.toBuilder().coscIssueDate(time.now().toLocalDate());
```

Comment:
shoul we rename it as "defendant Mark paid in full date" as its the date when defendant marked paid in full ?

Reply:
i believe thats different. this is when the cosc is generated - and we check that if the handler is marked paid in full. there are two dates introduced:

joMarkedPaidInFullIssueDate;
coscIssueDate;